libpod/container_internal_linux.go

```diff
@@ -668,11 +669,30 @@ func (c *Container) setupSystemd(mounts []spec.Mount, g generate.Generator) erro
 		}
 		g.AddMount(systemdMnt)
 	} else {
+		hostMounts, err := mount.GetMounts()
+		if err != nil {
+			return err
+		}
+
+		var hostMount *mount.Info
+		for _, m := range hostMounts {
+			if m != nil && m.Mountpoint == "/sys/fs/cgroup/systemd" {
+				hostMount = m
+				break
+			}
+		}
+
+		if hostMount == nil {
+			return errors.New("/sys/fs/cgroup/systemd is not mounted on the host")
```

Comment:
Can you make this `errors.Wrapf(define.ErrInternal, "/sys/fs/cgroup/systemd is not mounted on the host")` - we generally try to have all errors from Libpod wrap one of our defined errors whenever we can.